crossdc-consumer/src/main/java/org/apache/solr/crossdc/consumer/KafkaCrossDcConsumer.java

```diff
@@ -329,7 +329,7 @@ void processResult(ConsumerRecord<String,MirroredSolrRequest> record, IQueueHand
           log.trace("result=failed-resubmit");
         }
         metrics.counter("failed-resubmit").inc();
-        kafkaMirroringSink.submit(record.value());
+        kafkaMirroringSink.submit(result.newItem());
```

Comment:
this should do the trick :)
BTW, I don't like the name `_newItem` and this method suggests as if we were creating something new, which we're not. Why not `item` and `getItem()` ?

Reply:
I honestly have no clue. I started on this from Anshum's original dump that he extracted from some older code that did other things and integrated with other components. I don't understand much about the naming in that class, and since I haven't really had to touch before, it's just kind of survived. I'll make sure to refactor it in a cleanup issue.